share/picongpu/tests/Synchrotron_plugin/lib/requirements.txt

```diff
@@ -1,4 +1,5 @@
 os
-numpy
-scipy
-openpmd_api
+numpy==1.26.4
+scipy==1.11.4
+pathlib
```

Comment:
`pathlib` is part of the standard library, so we don't need to include it.

Reply:
Done

Reply:
The same with os, right?

Reply:
Yes